img/d/desmos.com.svg

```diff
@@ -0,0 +1 @@
14.03511 50.70182,14.9123c20.87722,0.87719 33.68426,-1.92983 54.38604,-11.92984c21.2281,-10.17545 35.26321,-21.75442 74.91239,-61.75447c47.36849,-47.54393 63.6843,-58.42113 87.36854,-58.42113c22.10529,0 36.31584,9.82458 89.29837,61.22816c49.29831,47.71936 63.85974,58.59657 90.17556,66.8422c22.10529,7.01755 52.28077,5.61404 73.68431,-3.50878c2.98246,-1.22807 2.98246,7.36843 2.63158,188.77219c-0.52632,183.50903 -0.70176,190.35114 -3.85965,197.01782c-8.07019,16.14037 -22.63161,24.386 -41.22813,23.5088l-11.22809,-0.52632l-5.96492,-22.80705c-51.40358,-196.84238 -116.14051,-372.45666 -174.21077,-472.28136l-12.45616,-21.57898l-7.89475,0c-9.29826,0 -23.68424,6.66668 -37.0176,17.36845l-8.94738,7.19299l10.70177,17.19301c29.8246,47.36849 65.965,129.82474 102.45628,232.80734c21.40354,60.87728 35.61408,107.19313 69.64922,228.59681l4.21053,14.38598l-293.68462,0l-293.50918,0l5.26317,-19.64915c16.84213,-63.33342 43.85971,-153.15811 61.92991,-205.78976c24.386,-71.57905 56.31587,-152.63179 72.63168,-184.21078l5.4386,-10.52633l-44.56147,-42.80708l-9.4737,19.64915c-40.87725,85.61415 -93.68434,233.85997 -133.15808,374.03561c-5.78948,20.35091 -12.10528,44.21059 -14.38598,53.15797c-2.10527,8.77194 -4.73685,16.14037 -5.96492,16.14037c-1.05263,0 -5.08773,-1.57895 -8.94738,-3.33334z"/><path d="m749.12385,339.12328c-20.00003,-3.33334 -34.73689,-14.03511 -77.01765,-55.78955c-44.03515,-43.50883 -57.71938,-54.91236 -78.24572,-64.91237c-19.47371,-9.64914 -32.80706,-13.15791 -50.00007,-13.15791c-17.36845,0 -32.9825,3.85965 -50.00007,12.28072c-21.57898,10.70177 -35.43865,22.28073 -74.56151,61.40359c-50.87726,50.87726 -63.50886,59.64921 -86.66679,59.64921c-12.28072,-0.17544 -25.26319,-4.73685 -38.77198,-14.03511c-4.91229,-3.1579 -25.26319,-22.28073 -45.6141,-42.28076c-40.70181,-40.17549 -47.36849,-45.96498 -64.21062,-57.01762c-20.87722,-13.50879 -41.05269,-20.00003 -63.50886,-20.00003c-17.89476,0 -26.49126,2.10527 -47.71936,12.28072c-18.59652,8.77194 -34.38601,20.35091 -58.2457,42.80708l-14.56142,13.68423l0,-118.24578c0,-128.07035 0,-128.42123 9.4737,-140.70195c2.45614,-3.33334 8.5965,-8.07019 13.68423,-10.52633l9.12282,-4.56141l367.71981,0l367.71981,0l9.12282,4.56141c5.08773,2.45614 11.22809,7.19299 13.68423,10.52633c9.64914,12.45616 9.4737,10.70177 9.4737,163.85988l0,141.75458l-10.87721,7.01755c-14.38598,9.12282 -28.77197,13.33335 -40.00006,11.40352zm-447.89536,-59.29833c22.10529,-28.59653 49.47375,-51.0527 69.47378,-57.3685c19.29827,-5.96492 37.0176,-3.33334 56.31587,8.24563c6.3158,3.85965 11.05265,5.61404 12.28072,4.73685c1.05263,-0.87719 7.01755,-5.78948 13.33335,-10.70177c6.3158,-5.08773 16.66669,-12.45616 23.33337,-16.31581l11.7544,-7.01755l-11.7544,-9.4737c-25.08775,-19.82459 -55.61411,-30.87724 -84.56152,-30.87724c-27.71934,0 -50.17551,7.19299 -75.78958,24.21056c-14.38598,9.4737 -43.68427,37.71935 -59.12289,57.01762l-10.70177,13.33335l20.52634,20.52634c11.22809,11.05265 21.05266,20.17547 21.57898,19.82459c0.70176,-0.17544 6.66668,-7.36843 13.33335,-16.14037z"/></g></svg>
```

Comment:
I think you got that logo from `desmos.com.au`...

Reply:
I took the page icon from `desmos.com`, but it was only a png, so I converted it and colored it with the green from the image (without the gradient).

Reply:
@sn00py1310 converting PNGs to SVGs rarely works well. I usually google for the company name and append `filetype:svg` in the search query.

Heres' what I found when I did that:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="#136936" d="m127 293 27-26c-20-43-37-88-52-133S74 43 61-3h377l-5 17-7 24a1005 1005 0 0 1-37 121l-16 43c-12 29-24 59-40 87l-14 23c7 10 22 18 33 16l11-18c21-37 37-76 52-116A1664 1664 0 0 0 475 2l1-5h39v264c-2 1-4-1-5-1-24-10-47-6-68 8-13 9-23 20-35 31l-16 15-21 18a36 36 0 0 1-42 2c-12-7-21-17-30-26l-28-28a96 96 0 0 0-39-23c-21-6-40-1-58 10-12 7-22 19-33 29l-35 32c-7 6-15 11-25 11-7 1-14-1-21-4a802 802 0 0 1-55-50l-7-7V-3h26c24 95 55 187 93 277 5 6 6 13 11 19ZM515 305v209H-3V330l12 12c12 11 24 21 39 28 23 11 45 8 65-4 15-8 27-20 39-32 10-9 19-20 30-29 5-5 11-8 17-11 14-6 26-3 37 5 12 9 23 21 33 32 13 12 25 25 40 34 20 12 41 16 63 7 12-4 21-11 31-19 14-12 26-25 39-38 7-7 14-14 23-19 12-7 24-7 37 0l13 9Zm-357 40 1 1c11 15 24 30 40 41 12 9 25 15 40 17 28 4 51-6 73-25-12-6-22-14-32-22l-3 2c-18 11-35 13-54 0-11-7-20-17-29-27l-10-14-26 27Z"/></svg>
```

Reply:
@Carlgo11 thanks for the tip

Reply:
> I think you got that logo from `desmos.com.au`...

I just wanted to point out that the logo was upside-down, but the joke fell flat, I guess. :man_shrugging:

Reply:
Oh, and I tried everything to reach it but the domain wasn't sold.